platform-controller/src/main/java/org/hobbit/controller/ExperimentManager.java

```diff
@@ -69,7 +66,7 @@
     /**
      * Default time an experiment has to terminate after it has been started.
      */
-    public static final long DEFAULT_MAX_EXECUTION_TIME = 20 * 60 * 1000;
+    public static final long DEFAULT_MAX_EXECUTION_TIME = Integer.parseInt(System.getProperty("MAX_EXECUTION_TIME", Integer.toString(MAX_EXECUTION_TIME)));
```

Comment:
Although I understand the elegance of this line, I see this a little bit sceptical because of the follwing three reasons:
1. Putting this in one line without a `try-catch` block might lead to a crashing platform controller if the user configures something that can not be parsed as `int`. You could define a static method that can be reused in other classes as well or you try out `Integer.getInt`. I would suggest going for the static method because of the other two reasons.

2. `Long` might be better than `Integer` since the time is measured in milliseconds.

3. Can you please make sure that this works as expected since I remember that `System.getProperty` and `System.getenv` do not always return the same values. For testing it locally, you could temporarily add a simple `main` method that writes the `DEFAULT_MAX_EXECUTION_TIME` that it got from the properties and start this method from command line after adding a `MAX_EXECUTION_TIME` as env value.